### PR TITLE
Restructuring howto and measurements

### PR DESCRIPTION
**Pull request by JoschD**

### CODE REVIEW

File: docs/measurements/methods/basic.md [hunk 1]
```diff
@@ -68,10 +68,10 @@ $$
 This section gives a brief overview over the K-Modulation method.
```
Comment (mihofer) [inline docs/measurements/methods/basic.md:68]: since `methods` is now it's own tab, the methods could be split in different subpages

**Comment by mihofer** [comment]
In the omc team part, one could also add a link to the mattermost channel(s)

File: docs/guis/developing/ide_install.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ There is also an [installation guide available][accsoft_eclipse_wiki]{target=_bl
 
 For people using pycharm it might make sense to use IntelliJ IDEA, however we do not yet know how to export a JAR (see also [GUI Releases](releases.md)).
 
-As both IDEs require [CommonBuildNextGeneration (CBNG)][cbng_wiki]{target=_blank} to resolve dependencies and make releases, one should either run these from somewhere in the Technical Network (e.g. from the `dev-server`) or mount the required paths via `sshfs` as described [here](../../resources/howto/setup.md#mounting-tn-resources-on-gn-machines).
+As both IDEs require [CommonBuildNextGeneration (CBNG)][cbng_wiki]{target=_blank} to resolve dependencies and make releases, one should either run these from somewhere in the Technical Network (e.g. from the `dev-server`) or [mount the required paths via `sshfs`][mount_tn].
```
Comment (fsoubelet) [inline docs/guis/developing/ide_install.md:8]: This should be [mounting_tn]{target=_blank}

File: docs/howto/setup/git.md [hunk 3]
```diff
@@ -165,18 +115,15 @@ Whenever you are pushing now any commits to the `master` branch, the CI/CD will
 *[GPN]: General Purpose Network, the main CERN network
 *[CBNG]: Common Build Next Generation
 *[LSA]: LHC Software Architecture
-*[AFS]: Andrew File System
```
Comment (fsoubelet) [inline docs/howto/setup/git.md:168]: Probably should not have been removed?

**Review by fsoubelet — DISMISSED** [review]
Few things here and there. I haven't tested all links but I'm okay making changes and fixing issues as we find them, this isn't software after all.

The `The OMC Team` section feels a bit empty now though.

File: docs/guis/developing/ide_install.md (hunk 2 above)
Comment (JoschD) [reply docs/guis/developing/ide_install.md:8]: I thought new tabs only for external links?

File: docs/measurements/methods/basic.md (hunk 1 above)
Comment (JoschD) [reply docs/measurements/methods/basic.md:68]: It's already organized in that way, but the methods are very short, so I left it as is for now

File: docs/measurements/physics/calibration.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+(\Delta D_{x}^{\phi})^{2} = (D^\phi_x)^2 + \left(\frac{1}{2}\frac{D^A_{N,x}}{\sqrt{\beta_{x}^{\phi}}}\Delta \beta^{\phi}_{x}\right)^{2}
+$$
+
+Calibration factors are then defined as[^RamaDispersionCalibration]:
+
+$$
+C_{x}^{A} = \frac{D^A_{N,x}\sqrt{\beta^{\phi}_x}}{D^A_x}
+$$
+
+!!! info "Hozirontal Plane Only"
```
Comment (mihofer) [inline docs/measurements/physics/calibration.md:56]: typo patrol

File: docs/measurements/physics/calibration.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
+
+# BPM Calibration
+
+Beam Position Monitors (BPM) are instruments used to measure the transverse position of the beam at a certain longitudinal position in the accelerator.
+However, the returned values are often off by a certain factor called calibration factor.
```
Comment (mihofer) [inline docs/measurements/physics/calibration.md:5]: maybe adding a simple formula here and/or a link to Dr. Carlier Msc Thesis, where there is a section on that https://cds.cern.ch/record/2057179/files/CERN-THESIS-2015-154.pdf?

**Comment by JoschD** [comment]
> In the omc team part, one could also add a link to the mattermost channel(s)
 and I forgot about this.. working on it

File: docs/measurements/physics/calibration.md (hunk 5 above)
Comment (JoschD) [reply docs/measurements/physics/calibration.md:5]: -> make an issue

File: docs/guis/developing/ide_install.md (hunk 2 above)
Comment (fsoubelet) [reply docs/guis/developing/ide_install.md]: You're right!

**Review by mihofer — APPROVED** [review]

**Review by fsoubelet — APPROVED** [review]
I'm happy with this

**Pull request merged**